src/website/views/components/data-table/_properties.pug

```diff
@@ -49,6 +49,7 @@ section.chi-table.chi-table__options.-bordered.-my--3
                 selectable?: <code>boolean | 'radio'</code>;
                 reserveExpansionSlot?: <code>boolean</code>;
                 truncation?: <code>boolean</code>;
+                printMode?: <code>PrintModes</code>;
```

Comment:
```suggestion
                printMode?: <code>'default' | 'printonly' | 'screenonly'</code>;
```
For consistency with `clientside` | `serverside`, let's follow the same casing.

Reply:
I'm not sure if `default` is descriptive enough. Any config might have some default value but we need to make it descriptive. In this case, we need to emphasize that `default` mode is both print and screen.